### PR TITLE
Update config.def.h: add XF86HomePage key, remove appmenu on click on…

### PR DESCRIPTION
**Pull request by ppenguin**
Add XF86HomePage key, remove appmenu on click on root window (very annoying for touchscreens, not a good default IMHO)

### CODE REVIEW

File: config.def.h [hunk 1]
```diff
@@ -451,7 +452,7 @@ static Button buttons[] = {
 	{ ClkStatusText,        MODKEY,         Button4,        spawn,          {.v = upbright } },
 	{ ClkStatusText,        MODKEY,         Button5,        spawn,          {.v = downbright } },
 	{ ClkRootWin,           MODKEY,         Button3,        spawn,          {.v = notifycmd } },
-	{ ClkRootWin,           0,              Button1,        spawn,          {.v = panther } },
+	// { ClkRootWin,           0,              Button1,        spawn,          {.v = panther } },
```
Comment (con-f-use) [inline config.def.h:455]: If you're gonna remove it, remove it. Don't leave the comment a dangling zombie for all eternity :wink:

Comment (ppenguin) [reply config.def.h]: Good point :wink:  done.

**Comment by paperbenni** [comment]
Can you elaborate on why you think it's a bad default?
I know that some people have the habit of clicking around the desktop when thinking, but it's not a good habit anyway and having it like this improves explorability. 
The touchscreen thing interests me. Are you using instantOS on a device with a touchscreen?
Wouldn't you use tiling on these anyway?

**Comment by ppenguin** [comment]
> Can you elaborate on why you think it's a bad default?

If you use gestures on a touch screen you will often see that they are incorrectly registered, e.g. a 2-finger swipe is interpreted as a 1-finger click (I'd argue this may happen as much as 30% of the time, especially when you're used to mobile phone sensitivity. If the huge appmenu pops up for any of those times, and you can only make it disappear by pressing ESC (which you can't because no keyboard in tablet mode) or by touching the root window again, which is now a tiny gap between the screen edge and the screen-filling appmenu, this is absolutely unworkable.

The disadvantage of this behaviour is IMHO huge compared to the "advantage" of the current default behaviour. Or from a different angle: it's counter-intuitive that a minor, possibly not-so-deliberate action as clicking on the root-window fills your screen in a way that you can only undo by pressing a specific key (ESC) or clicking on a tiny gap (without launching an app).

Another argument is that I feel it's not good (inconsistent) that the behaviour of the desktop makes a difference between when it's empty or has an app open; this is another burden for the user to tune his reflexes not only to key-bindings but **additionally** to context.

> The touchscreen thing interests me. Are you using instantOS on a device with a touchscreen?

Indeed I installed this on a convertible, actually before my desktop, since I got hooked on tiling WMs recently (on my desktop, I started with awesomewm first). I actually found instantOS because I was looking for a tiling WM which at the same time would be touchscreen friendly, and figured instantWM to be a good match. I might add after using it a while: it **is** a very good match!

I might also add that overall the architecture of instantOS is pretty good IMO given its objective: mostly scripted and therefore easy to adapt by power users, mostly good defaults, fitting with linux philosophy (no built-ins, everything achieved by spawning standard utils) and keyboard friendly (very important to be a good alternative for other tiling WMs!). So kudos for that!

**Comment by ppenguin** [comment]
Some extra (off-topic) info:

On my desktop I just installed the `instant*` packages on top of my existing Manjaro install, that worked pretty well

On my convertible I used the beta-5 installer, after wiping an install I did with Manjaro Architect. 
The installer still needs **a lot of work** and, especially for the target group (power users), should make it at least possible to configure things like *LVM-on-LUKS* or similar.  I wonder if it isn't easier (i.e. better spent energy and resources) to base it on existing installer (actually Manjaro Architect was quite good, though I haven't checked how it works under the hood). A quick look at the installer scripts of instantOS didn't instill a lot of confidence (no offense), because it appears to have a lot of "patch-work" between many scripts, for different use-cases. I think this is very hard to get right, especially in terms of robustness and maintainability.

**Comment by con-f-use** [comment]
I vote to disable the "click on desktop opens app launcher" behavior, too. It always befuddles me, when I do it first, I don't think it adds much in terms of discoverability. If you're dead set on discoverability, maybe have an empty top bar (i.e. no windows in the currently active tab(s)) say "Click here to open the Applications Menu --->".

**Pull request closed without merging**